blocks/file-blocks/code/index.tsx

```diff
@@ -87,26 +89,13 @@ export default function (props: FileBlockProps) {
   const editorRef = React.useRef<HTMLDivElement>(null);
   const viewRef = React.useRef<EditorView>();
 
-  if (viewRef.current) {
-    const view = viewRef.current;
-    const doc = view.state.doc.sliceString(0);
-    if (doc !== content) {
-      view.dispatch({
-        changes: { from: 0, to: doc.length, insert: content },
-        // mark the transaction remote so we don't call `onUpdateContent` for it below
```

Comment:
TIL - feels like we need a "tips & tricks" doc for codemirror, it's such an extensive API

Reply:
yes!